### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (.btn-icon-dropdown)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Problem-Roulette#2]
#### Issue
`Buttons must have discernible text` ([button-name](https://accessibilityinsights.io/info-examples/web/button-name))
#### Target application
[Problem Roulette](https://staging.problemroulette.ai.umich.edu/StudentDashboard/)
#### Element path
.btn-icon-dropdown
#### Snippet
`<button aria-haspopup="true" aria-owns="dropdown_option" class="btn-icon-dropdown online-status"> <div class="status-dot active"></div> <div class="user-account"></div> <span></span></button>`
#### Environment
Chrome version 114.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/7abf8830-4707-4020-9ec5-8c607793c4be)

**Comment by zcjess** [comment]
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/02b27d9e-2faf-4c32-a49f-79f7a79df49e)

**Comment by zcjess** [comment]
Title: WCAG 4.1.2: Select element must have an accessible name (select)
Tags: Accessibility, WCAG 4.1.2, select-name

Issue: Select element must have an accessible name (select-name - https://dequeuniversity.com/rules/axe/4.6/select-name?application=msftAI)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/console/ReviewStudentFeedback

Element path: select

Snippet: <select name="page-items" class="page-items"><option value="25">25</option> <option value="50">50</option> <option value="100">100</option> <option value="200">200</option></select>



How to fix | Fix ONE of the following:Form element does not have an implicit (wrapped) <label>Form element does not have an explicit <label>aria-label attribute does not exist or is emptyaria-labelledby attribute does not exist, references elements that do not exist or references elements that are emptyElement has no title attributeElement's default semantics were not overridden with role="none" or role="presentation"
-- | --


====
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/514013cc-8c26-40b1-88a5-3121fc4b0d9b)

![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/446d3d17-9299-4cdc-a771-92b6d539ceb8)

**Comment by zcjess** [comment]
Title: WCAG 4.1.2: Select element must have an accessible name (select)
Tags: Accessibility, WCAG 4.1.2, select-name

Issue: Select element must have an accessible name (select-name - https://dequeuniversity.com/rules/axe/4.6/select-name?application=msftAI)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/console/AES220/CourseAnalytics/

Element path: select

Snippet: <select class="questionset-filter topic-select"> <option data-accuracy="[{'accuracy': 25.0, 'answered': 8}]">"In-Class" Quiz 3 PRACTICE</option> </select>

Environment: Chrome version 114.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/4fd629f7-b35b-49b5-a65b-e25dd168f5a2)
https://staging.problemroulette.ai.umich.edu/console/AES220/CourseAnalytics/

**Comment by zcjess** [comment]
Title: WCAG 4.1.2: Buttons must have discernible text (.btn-icon-dropdown)
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Buttons must have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/AES220/SessionManager/156390/

Element path: .btn-icon-dropdown

Snippet: <button aria-haspopup="true" aria-owns="dropdown_option" class="btn-icon-dropdown online-status"> <div class="status-dot"></div> <div class="user-account"></div> <span></span></button>

Environment: Chrome version 114.0.0.0

====

Title: WCAG 4.1.2: Buttons must have discernible text (.btn-danger)
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Buttons must have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

Target application: Problem Roulette - https://staging.problemroulette.ai.umich.edu/AES220/SessionManager/156390/

Element path: .btn-danger

Snippet: <button class="btn btn-danger btn-small chatbox-close-btn" type="button"><span class="glyphicon glyphicon-remove"></span></button>

Environment: Chrome version 114.0.0.0

====

![image](https://github.com/zcjess/Problem-Roulette/assets/90011962/f96c8ce7-df3b-4a8c-8d68-ecff50a89719)
